pkgs/applications/graphics/imgp/default.nix

```
@@ -0,0 +1,28 @@
+{ lib, fetchFromGitHub, buildPythonApplication, pillow, imgp }:
+
+buildPythonApplication rec {
+  pname = "imgp";
+  version = "2.7";
+
+  src = fetchFromGitHub {
+    owner = "jarun";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "13r4fn3dd0nyidfhrr7zzpls5ifbyqdwxhyvpkqr8ahchws7wfc6";
+  };
+
+  propagatedBuildInputs = [ pillow ];
+
+  installFlags = [
+    "DESTDIR=$(out)"
+    "PREFIX="
+  ];
+
```

just to ensure that the package is able to import itself and dependencies correctly:
```suggestion

  checkPhase = ''
    $out/bin/impg --help
  '';
```

it's better than nothing

---

Added.